app/Models/Feed.php

```diff
@@ -104,9 +111,12 @@ public function keepHistory() {
 	public function ttl() {
 		return $this->ttl;
 	}
+	public function mute() {
+		return $this->mute;
```

Comment:
Do we need a variable, or is it always `return $this->ttl < self::TTL_DEFAULT;`
I have become confused along the way: I do not have a strong feelings either way, but I though the `muted` state would be a kind of meta property to quickly change TTL and priority at once.

Reply:
It's always ```return $this->ttl < self::TTL_DEFAULT;```.
Muting the refresh won't change the display. I can force that behavior the same way I did for the ttl. So the mute is not a meta property at the moment.